taigit/src/components/Overview.js

```diff
@@ -4,12 +4,17 @@ import {Doughnut, Line} from 'react-chartjs-2';
 import { saveToLocalStorage, getFromLocalStorage } from '../utils/utils';
 import { WidthProvider, Responsive } from "react-grid-layout";
 import colors from '../styles/colors';
+import { Bar } from 'react-chartjs-2';
+import { selectUserTaskDistributionChartData } from '../reducers';
+import { connect } from 'react-redux';
+import { grabTaigaData, grabSprintStats, grabTaskStats } from '../actions/taigaActions';
```

Comment:
I believe you can remove the import of _grabTaigaData_ and _grabSprintStats_, since we only need to call _grabTaskStats_ here